.github/workflows/end-to-end-tests.yml

```
@@ -56,7 +56,7 @@ env:
 jobs:
   # Runs the end-to-end test suite.
   e2e-tests:
-    name: Test with SCRIPT_DEBUG ${{ matrix.LOCAL_SCRIPT_DEBUG && 'enabled' || 'disabled' }}
+    name: ${{ matrix.label }}
```

Nit: As we don't use the `label` else where can we use label directly here?

```suggestion
    name: E2E Tests
```

---

This is actually a trick to get the UI in GitHub Actions to do what we want. A matrix value is needed in the `name` directive for proper grouping in the GitHub UI.